lib/jekyll/watcher.rb

```diff
@@ -41,10 +42,10 @@ def watch(options, site = nil)
     # TODO: shouldn't be public API
     def build_listener(site, options)
       Listen.to(
-        options['source'],
+        @source,
         :ignore => listen_ignore_paths(options),
         :force_polling => options['force_polling'],
-        &(listen_handler(site))
+        &(listen_handler site)
```

Comment:
rubocop had raised an offense for this locally..
perhaps it wont, at the version locked to, for this plugin.. I'll update..